sqlstore/src/main/resources/db/migration/V0028__model_refactor.sql

```diff
@@ -85,9 +85,45 @@ DROP TABLE SampleAnalyte;
 DELETE FROM SampleClass WHERE sampleCategory = 'Analyte';
 ALTER TABLE SampleClass DROP COLUMN isStock;
 
--- CREATE SYNTHETIC TISSUES HERE
-
 --StartNoTest
+-- CREATE SYNTHETIC TISSUES
+DROP PROCEDURE IF EXISTS createsynthetic;
+DELIMITER //
+CREATE PROCEDURE createsynthetic()
+BEGIN
+  DECLARE done INT DEFAULT FALSE;
+  DECLARE badId bigint(20);
+  DECLARE newId bigint(20);
+  DECLARE badChildren CURSOR FOR SELECT sampleId FROM SampleAdditionalInfo WHERE sampleClassId IN (
+    SELECT svr.childId
+    FROM SampleValidRelationship svr
+    JOIN SampleClass parent ON parent.sampleClassId = svr.parentId
+    JOIN SampleClass child ON child.sampleClassId = svr.childId
+    WHERE parent.sampleCategory = 'Identity' AND child.sampleCategory <> 'Tissue');
+  DECLARE CONTINUE HANDLER FOR NOT FOUND SET done = TRUE;
+
+  OPEN badChildren;
+
+  read_loop: LOOP
+    FETCH badChildren INTO badId;
+    IF done THEN
+      LEAVE read_loop;
+    END IF;
+    INSERT INTO Sample(name, description, securityProfile_profileId, sampleType, receivedDate, project_projectId, scientificName, taxonIdentifier, lastModifier, emptied) SELECT 'TEMP', description, securityProfile_profileId, sampleType, receivedDate, project_projectId, scientificName, taxonIdentifier, lastModifier, emptied FROM Sample WHERE sampleId = badId;
```

Comment:
Alias never gets set. While there isn't a great way of doing this without the naming scheme, maybe it could be set to something similar to the name field so it is at least not null